Include logging dependencies

From: Lars Francke <git@lars-francke.de>

This adds dependenciesr required for use of the XmlLayout for logging
---
 standalone-metastore/pom.xml |   20 ++++++++++++++++++--
 1 file changed, 18 insertions(+), 2 deletions(-)

diff --git a/standalone-metastore/pom.xml b/standalone-metastore/pom.xml
index 3b35b0ba71..702ff61892 100644
--- a/standalone-metastore/pom.xml
+++ b/standalone-metastore/pom.xml
@@ -73,7 +73,7 @@
     <guava.version>19.0</guava.version>
     <hadoop.version>3.1.4</hadoop.version>
     <hikaricp.version>2.6.3</hikaricp.version>
-    <jackson.version>2.12.7.2</jackson.version>
+    <jackson.version>2.12.7.20240502</jackson.version>
     <javolution.version>5.5.1</javolution.version>
     <junit.version>4.11</junit.version>
     <libfb303.version>0.9.3</libfb303.version>
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>${jackson.version}</version>
     </dependency>
     <dependency>
       <groupId>com.github.joshelser</groupId>
@@ -272,6 +271,11 @@
       <artifactId>log4j-core</artifactId>
       <version>${log4j2.version}</version>
     </dependency>
+    <dependency>
+      <!-- Optional log4j dependency to be able to use the XmlLayout -->
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-xml</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.apache.thrift</groupId>
       <artifactId>libfb303</artifactId>
@@ -350,6 +354,18 @@
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+          <groupId>com.fasterxml.jackson</groupId>
+          <artifactId>jackson-bom</artifactId>
+          <version>${jackson.version}</version>
+          <scope>import</scope>
+          <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <profiles>
     <profile>
       <id>thriftif</id>
